extensions/wikia/AuthModal/js/AuthModal.js

```diff
@@ -17,12 +19,31 @@ define('AuthModal', ['jquery', 'wikia.window'], function ($, window) {
 		$(window).on('message.authPopUpWindow', function (event) {
 			var e = event.originalEvent;
 
-			if (typeof e.data !== 'undefined' && e.data.isUserAuthorized) {
+			if (typeof e.data === 'undefined') {
```

Comment:
Yeah, this check wasn't the best at the beginning, I think checking if e.data is of not of type object would make more sense? Or if it's truthy.